frontend_tests/node_tests/password.js

```diff
@@ -2,11 +2,16 @@ add_dependencies({
     zxcvbn: 'node_modules/zxcvbn/dist/zxcvbn.js',
 });
 
+set_global('i18n', {
```

Comment:
Thanks to @adnrs96, we now have a built-in helper for `i18n` stubbing!  So you should be able to just say `var i18n = global.i18n` here to appease the linker.